src/mutation.ts

```diff
@@ -380,11 +380,19 @@ export class Mutation {
    * @private
    */
   static parseColumnName(columnName: string): ParsedColumn {
-    const parts = columnName.split(':');
+    const colonIdx = columnName.indexOf(':');
+
+    if (colonIdx === -1) {
+      // columnName does not contain ':'
+      return {
+        family: columnName,
+        qualifier: null,
```

Comment:
I recommend setting `qualifier` to `undefined` so that this code works the same way as before when the string doesn't contain a colon.

Reply:
I tried that too, however the `ParsedColumn` interface does not declare `undefined` as an acceptable type for `qualifier` at https://github.com/googleapis/nodejs-bigtable/blob/main/src/mutation.ts#L35.

Adding `undefined` here would solve this:

```typescript
export interface ParsedColumn {
  family: string | null;
  qualifier: string | null | undefined;
}
```